### DISCUSSION TITLE
【robots.txt】 页面显示不正确

### DISCUSSION THREAD

**Discussion by etjec4** [tangly1024/NotionNext#1999] [Q&A]
**描述bug**
LINK 变量已经设置，4.2.0版本，但 robots.txt 页面显示不正确，

![image](https://github.com/tangly1024/NotionNext/assets/144890343/f9464dad-74b4-4ad1-8c2d-3ba651b2bef5)

**环境**

- 【必填】NotionNext版本 [4.2.0]
- 【必填】主题 [hexo]
- 【可选】操作系统: [windows]
- 【可选】浏览器 [chrome]

**补充说明**
如何配置才能正确显示？

**Comment by tangly1024** [comment]
Vercel后台 配置 NEXT_PUBLIC_LINK 环境变量，并且重新redeploy

```
  LINK: process.env.NEXT_PUBLIC_LINK || 'https://tangly1024.com', // 网站地址
```

redeploy时，不要勾选 ‘use existed cache’
